tests/test-files/good/AllSyntax/0.18/Literals.json

```diff
@@ -169,7 +183,21 @@
             "type": "Definition"
         },
         {
-            "expression": "TODO: Expr",
+            "expression": {
+                "module": "String",
+                "sourceLocation": {
+                    "end": {
+                        "col": 22,
+                        "line": 34
+                    },
+                    "start": {
+                        "col": 5,
+                        "line": 34
+                    }
+                },
+                "type": "StrLiteral",
+                "value": "a b'`<>{}/\u06a5\ud83d\ude00\u3041\u2f83\ud835\udfdf"
```

Comment:
Do things work correctly when something that becomes an escaped unicode character is immediately followed by a hex digit?  Even if so, it looks like an example of that happening is missing from the `Literals.elm` test file

Reply:
Apparently it works...  though the choice in escaping is a little weird imo.  This literal string `😀0😀9😀a😀F` results in the following json respresentation: `"\ud83d\ude000\ud83d\ude009\ud83d\ude00a\ud83d\ude00F"`

But I guess the details of the escaping don't really matter, since the consumers of the API can be expected to be using a compliant JSON parser.